templates/CONTRIBUTING.md

```diff
@@ -0,0 +1,131 @@
+# Contributing
+
+---
+
+COMMENT TO REMOVE
+
+Be aware of XXX in sentences and links: they need to be replaced by the name of the repository.
+
+---
+
+First, thank you for contributing to MeiliSearch! The goal of this document is to provide everything you need to start contributing to MeiliSearch.
+
+<!-- MarkdownTOC autolink="true" style="ordered" indent="   " -->
+
+- [Assumptions](#assumptions)
+- [How to Contribute](#how-to-contribute)
+- [Development Workflow](#development-workflow)
+- [Git Guidelines](#git-guidelines)
+
+<!-- /MarkdownTOC -->
+
+## Assumptions
+
+1. **You're familiar with [GitHub](https://github.com) and the [Pull Request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)(PR) workflow.**
+2. **You've read the MeiliSearch [docs](https://docs.meilisearch.com) and the [README](/README.md).**
```

Comment:
```suggestion
2. **You've read the MeiliSearch [documentation](https://docs.meilisearch.com) and the [README](/README.md).**
```

Which README are you referring to? Meilisearch/meilisearch or the README of the project they are planning to contribute to?

Reply:
It will be the readme of the SDK repo